pulumi/folio/classes/FolioDeployment.ts

```diff
@@ -1,37 +1,23 @@
-import * as k8s from "@pulumi/kubernetes";
-import * as eks from "@pulumi/eks";
-
 /**
- * Represents details about a FOLIO kubernetes deployment, including the cluster
- * to deploy to, the namespace, the tenant, etc.
+ * Represents details about a FOLIO kubernetes deployment.
  */
 export class FolioDeployment {
     tenantId: string;
 
     deploymentConfigurationFilePath: string;
 
-    loadReferenceData: boolean;
-
-    loadSampleData: boolean;
-
     okapiUrl: string;
 
     containerRepository: string;
 
     constructor(tenantId: string,
         releaseFilePath: string,
-        loadReferenceData: boolean,
-        loadSampleData: boolean,
```

Comment:
Note the `FolioDeployment` no longer controls these items. They're handled in `register-modules.sh`.